ARCHITECTURE.md

```diff
@@ -41,4 +42,27 @@ For more details about the models we use in Scanner see web-monitoring-db's [API
 - web-monitoring-processing and web-monitoring-versionista-scraper are manually deployed to servers on AWS.
 - We plan to re-deploy these to AWS as a Kubernetes cluster, **but that doesn’t exist yet.**
 
+## Current Workflow
 
+| Diagram key | What happens | What does this | How | Criteria |
+|-|-------------|-|-------|---------|
+| A | Versionista is scraped | [web-monitoring-versionista-scraper](https://github.com/edgi-govdata-archiving/web-monitoring-versionista-scraper) | Data is scraped from Versionista, uploaded to S3, formatted, and [POST](https://api.monitoring.envirodatagov.org/#/imports/post_imports_)ed to [-db](https://github.com/edgi-govdata-archiving/web-monitoring-db) | Runs on a cron job, formats per [GH issue comment](https://github.com/qri-io/walk/issues/16#issuecomment-437785099) |
+|B| IA's Wayback Machine is queried for imports | [web-monitoring-processing](https://github.com/edgi-govdata-archiving/web-monitoring-processing) - this [PR](https://github.com/edgi-govdata-archiving/web-monitoring-processing/pull/174) | Data is pulled from the IA API, formatted, and [POST](https://api.monitoring.envirodatagov.org/#/imports/post_imports_)ed to [-db](https://github.com/edgi-govdata-archiving/web-monitoring-db) | Runs on a cron job, formats per [GH issue comment](https://github.com/qri-io/walk/issues/16#issuecomment-437785099) |
+|C| New metadata arrives at [-db](https://github.com/edgi-govdata-archiving/web-monitoring-db) | Scripts running on cron jobs that do ETL via scrapes or APIs (These scripts: [Versionista scraper](https://github.com/edgi-govdata-archiving/web-monitoring-versionista-scraper), [Wayback Machine importer](https://github.com/edgi-govdata-archiving/web-monitoring-processing))| [POST](https://api.monitoring.envirodatagov.org/#/imports/post_imports_) to [/api/v0/imports](https://api.monitoring.envirodatagov.org/#/imports/post_imports_) as JSON array or newline-delimited JSON stream (stream preferred) | Contains:<ul><li> a URL for where to retrieve the raw response body of the version</li><li> a SHA-256 hash of that data</li></ul> Example data in [GH issue comment](https://github.com/qri-io/walk/issues/16#issuecomment-437785099)|
+|D| Determination is made whether or not to download the data | [-db](https://github.com/edgi-govdata-archiving/web-monitoring-db) (specifics useful?) | If the URL is in an acceptable, publicly readable location |  Acceptable location checked [per .env](https://github.com/edgi-govdata-archiving/web-monitoring-db/blob/30d976676935f5f621e18285016a137c8b55a0d8/.env.example#L26-L27) |
```

Comment:
Is it worth adding a line (or putting it in with C or D, or maybe replacing F) that metadata gets stored in a database, and then for raw version bodies, we do this bit about determining whether to download the data?